apps/docs/components/blocks/RatingForms.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: DefaultLayout
+hideBreadcrumbs: true
+description: The e-commerce rating and review solution provides a platform for users to rate their e-commerce experiences. Users can assign ratings based on their satisfaction, product quality, delivery experience, and other relevant factors. These ratings offer valuable quantitative information that helps other potential buyers gauge the overall reputation and quality of products and services offered on the platform.
+hideToc: true
+---
+# Rating Forms
+
+{{ $frontmatter.description }}
+
+## Product Rating
+
+The Product Rating block serves as a basic yet effective solution for implementing a 1-5 stars rating system commonly used in eCommerce platforms. It enables users to easily express their opinion about a particular item or experience.
+
+It allows for customization through the use of RatingButton basic component. This feature permits users to provide more specific ratings, enhancing the granularity of their feedback.
```

Comment:
I don't think I understand this section. I think it's about customizing stars using the RatingButton, but I don't get the other part about "granularity". Could you rephrase this section?

Reply:
As I'm not an author I've asked Jacek about clarification on jira

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2849/commits/849575865c779fac693813b0589ad02389e9c134